CPAC/utils/ndmg_utils.py

```diff
@@ -183,8 +189,9 @@ def cor_graph(self, timeseries, attr=None):
         """
         import numpy as np
 
-        timeseries[0]
+        ts = timeseries[0]  # noqa: F841
```

Comment:
This `ts` isn't used anywhere. Do we care to assign it just to annotate, or should we drop this line?

Reply:
It's fine for now. Your comments with idea stubs in this PR is a pretty nice list, so we can revisit this when we revisit the other ideas as well.